client/me/purchases/purchases-list/index.jsx

```diff
@@ -30,6 +30,11 @@ import {
 	isFetchingUserPurchases,
 } from 'state/purchases/selectors';
 import { recordTracksEvent } from 'state/analytics/actions';
+import userFactory from 'lib/user';
+import NoSitesMessage from 'components/empty-content/no-sites-message';
+
+const user = userFactory();
+const userHasNoSites = () => user.get().site_count <= 0;
```

Comment:
Rather pulling in lib/user here, let's connect to our redux store with this selector:

https://github.com/Automattic/wp-calypso/blob/3a0e026843cca5d4fa0a1ea14ccb371765bd6f99/client/state/current-user/selectors.js#L81-L94